src/vs/editor/contrib/codeAction/browser/codeActionContributions.ts

```diff
@@ -40,8 +40,8 @@ Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfigurat
 		'editor.codeActionWidget.includeNearbyQuickfixes': {
```

Comment:
Shouldn't the setting name casing also be changed to be `includeNearbyQuickFixes`?
I think that will fix the spacing and casing in the Settings editor

Reply:
good catch - i thought i had changed that but must have reverted it at some point while i was pushing